src/__generated__/Accept_order.graphql.ts

```diff
@@ -24,6 +24,11 @@ export type Accept_order = {
     readonly " $fragmentRefs": FragmentRefs<"TransactionDetailsSummaryItem_order" | "ArtworkSummaryItem_order" | "ShippingSummaryItem_order" | "CreditCardSummaryItem_order">;
     readonly " $refType": "Accept_order";
 };
+export type Accept_order$data = Accept_order;
+export type Accept_order$key = {
+    readonly " $data"?: Accept_order$data;
+    readonly " $fragmentRefs": FragmentRefs<"Accept_order">;
+};
```

Comment:
All of these additions are to support the experimental hooks (which needs `react-relay` to be installed from 1pm’s `experimental` dist-tag).